_includes/docs/latest/ajax-api.html

```diff
@@ -1045,8 +1045,8 @@ <h3 id="api-schedule-a-flow">Schedule a Flow</h3>
       <td>The name of the flow.</td>
     </tr>
     <tr>
-      <td>scheduleTime</td>
-      <td>The time to schedule the flow. Example: 12,00,pm,PDT</td>
+      <td>scheduleTime(with timezone)</td>
+      <td>The time to schedule the flow. Example: 12,00,pm,PDT (Unless UTC is specified, Azkaban will take current server's default timezone instead)</td>
```

Comment:
@HappyRay 
Yes. The code logic [here](https://github.com/azkaban/azkaban/blob/master/azkaban-webserver/src/main/java/azkaban/webapp/servlet/ScheduleServlet.java#L694) might be not correct.